Makefile

```diff
@@ -65,6 +65,7 @@ generate-go: tool-install
 lint-go: tool-install
 	go mod tidy
 	export pkgs="`go list -f '{{.Dir}}' ./... | grep -v /proto/`" && echo "$$pkgs" | xargs go vet -vettool=$(TOOL_BIN)/combined
+	GOGC=50 $(TOOL_BIN)/golangci-lint run --config=./etc/.golangci.yaml || true
```

Comment:
I think what you have now is great since it's definitely an improvement. But, I'm wondering if we can somehow change the lint CI job to run this command (no --fix), that catches more errors, and fail the job if it returns a non-zero exit code. It seems like that would be both more informative in its printout and help us catch more lint issues?

Reply:
Can we make the local `make lint` run 2 and 3, and GitHub CI use a new make target that only runs 3?